lib/Transforms/Instrumentation/KleeInternalize.cc

```diff
@@ -239,13 +243,13 @@ namespace
 
     bool runOnModule (Module &M)
     {
+      m_module = &M;
       declareKleeFunctions(M);
       internalizeVariables(M);
 
       for (Function &F : M)
       {
-        if (shouldInternalize (F)) defineFunction (F);
```

Comment:
Is that because now the function is defined at a call site, so that each call site has a different definition?